trade-finance-manager-ui/templates/index.njk

```diff
@@ -2,7 +2,13 @@
 <html lang="en" class="govuk-template ">
   <head>
     <meta charset="utf-8" />
-    <title{% if pageTitleLang %} lang="{{ pageTitleLang }}"{% endif %}>{% block pageTitle %}{% endblock %} - TFM</title>
+    <title
+      {% if pageTitleLang %}
+        lang="{{ pageTitleLang }}"
+      {% endif %}
+      >
+      {% block pageTitle %}{% endblock %}, Trade Finance Manager, gov.uk
```

Comment:
New line `endBlock`?

Reply:
After more research we decided to move `<title>` back to single line. It matches gov.uk templates and avoids white space in the <title>.